utils/config_stores.py

```diff
@@ -0,0 +1,226 @@
+# std
+import glob
+import imp
+import inspect
+import itertools
+import logging
+import simplejson as json
+from os import path
+
+# 3p
+from consul import Consul
+from etcd import EtcdKeyNotFound
+from etcd import Client as etcd_client
+from urllib3.exceptions import TimeoutError
+
+
+log = logging.getLogger(__name__)
```

Comment:
you can't use that logger. As it has no handler.

You should use `logging.getLogger('collector')